Report/ReportElements/MaxDrawdownReportElement.cs

```diff
@@ -42,7 +45,24 @@ public MaxDrawdownReportElement(string name, string key, BacktestResult backtest
         /// </summary>
         public override string Render()
         {
-            return _backtest?.TotalPerformance?.PortfolioStatistics?.Drawdown.ToString("P1") ?? "-";
+            if (_live == null)
+            {
+                return _backtest?.TotalPerformance?.PortfolioStatistics?.Drawdown.ToString("P1") ?? "-";
+            }
+
+            var backtestEquityPoints = new Series<DateTime, double>(ResultsUtil.EquityPoints(_backtest));
+            var liveEquityPoints = new Series<DateTime, double>(ResultsUtil.EquityPoints(_live));
+
+            var backtestDrawdownGroups = new DrawdownCollection(backtestEquityPoints, 5);
+            var liveDrawdownGroups = new DrawdownCollection(liveEquityPoints, 5);
```

Comment:
Wonder why fixed `5` periods? Think it would be nice to see some tests for this guy too